docs/self-managed/operational-guides/update-guide/840-to-850.md

```diff
@@ -19,3 +19,15 @@ The broker health check routes have moved, and the old routes are now deprecated
 | http://{zeebe-broker-host}:9600/startup | **http://{zeebe-broker-host}:9600/actuator/health/startup**   |
 
 Please migrate to the new routes in your deployments. **If you're using the official Helm charts, then you don't have to do anything here.**
+
+### Changes to exported records
+
+The `UserTask` events like `UserTask:CREATED` don't export the string value attributes `candidateUsers` and `candidateGroups` anymore.
+As a replacement, user task events now feature the string array attributes `candidateUsersList` and `candidateGroupsList`.
+Custom exporters using these events must be modified accordingly.
```

Comment:
🔧 The `candidateUsers` and `candidateGroups` are configured in the Modeler using the properties panel. I suggest using `properties` over `attributes` (but it's not a strong opinion)

Reply:
No strong opinion here either on that. Your reasoning makes sense to me and it makes it a bit more consistent with how users model this. I'll adjust it 👍